userspace/engine/formats.cpp

```diff
@@ -44,7 +44,6 @@ std::string falco_formats::format_event(gen_event *evt, const std::string &rule,
 	formatter = m_falco_engine->create_formatter(source, format);
 
 	// Format the original output string, regardless of output format
```

Comment:
can we remove this comment? it seems we don't need it anymore

Reply:
Sure @Andreagit97, I removed the comment and committed the changes. Is everything else fine? Please let me know if any more changes are required in PR 😀